internal/translation/deployment/deployment.go

```diff
@@ -16,7 +16,7 @@ import (
 
 type AtlasDeploymentsService interface {
 	ListClusterNames(ctx context.Context, projectID string) ([]string, error)
-	ListDeploymentConnections(ctx context.Context, projectID string) ([]Connection, error)
+	ListDeploymentConnections(ctx context.Context, projectID string, isGov bool) ([]Connection, error)
```

Comment:
I do not think this change is needed here. The implementation can remember on its own if it is gov or not. It is a object field, rather than a parameter.

Reply:
Yes, good idea. Let me quickly refactor this